docs/en/sql-reference/aggregate-functions/reference/initializeAggregation.md

```diff
@@ -0,0 +1,31 @@
+---
+toc_priority: 150
+---
+
+## initializeAggregation {#initializeaggregation}
+
+Initializes aggregation for your input lines.
+Use it for tests or to process columns of types `AggregateFunction` and `AggregationgMergeTree`.
+
+**Syntax**
+
+``` sql
+initializeAggregation(input_rows_count);
```

Comment:
but in the example I see `initializeAggregation('uniqState', number % 3)`, it is more than just `input_rows_count`.

and you've lost the `Parameters` section.